libs/web/test-environment/src/lib/test-environment.component.ts

```diff
@@ -4,7 +4,14 @@ import { Event, NavigationEnd, Router } from '@angular/router';
 import { Subscription } from 'rxjs';
```

Comment:
Need to talk about all changes

Reply:
Still the calls are not synchronized

Reply:
That's why i didn't wrote that it's fixed 👍 
Me and Fabian wanted to look into it today and try out the `concatMap()` Operator from `rxjs`.

Reply:
Fixed in 362a88b and added more precise error-message in 3761f22.

Reply:
Should be fixed in 8a935cb.